frontend/src/metabase/dashboard/components/QuestionPicker/QuestionPicker.tsx

```diff
@@ -68,9 +69,8 @@ function QuestionPickerInner({
     [databases],
   );
 
-  const onNewQuestion = (type: "native" | "notebook") =>
```

Comment:
~"notebook" is no longer valid here I assume? since we only have onNewNativeQuestion now.~ nevermind, I saw it lifted to another file.